internal/constellation/helm/helm_test.go

```diff
@@ -194,7 +194,7 @@ func TestHelmApply(t *testing.T) {
 				awsLbVersion = *tc.clusterAWSLBVersion
 			}
 
-			certManagerVersion := "v1.10.0" // current version
+			certManagerVersion := "v1.13.2" // current version
```

Comment:
```suggestion
			certManagerVersion := "v1.12.6" // current version
```
Not sure if this is relevant for the test